utils/processInput.js

```diff
@@ -13,72 +13,73 @@ const generateHTMLFile = require("./generateHTML");
  * @param {string} HTMLlanguage - Language used when generating HTML
  */
 async function processInput(inputPath, stylesheetURL, HTMLlanguage) {
-    if (!existsSync(inputPath)) {
-        console.log("File/folder's path does not exist.");
-        return process.exit(1);
-    }
+  if (inputPath === "") {
```

Comment:
I don't think we need this `if` statement here, since if the user doesn't specify any path, that means the path doesn't exist.